dashboards-observability/common/constants/shared.ts

```diff
@@ -158,6 +158,7 @@ export interface DefaultChartStylesProps {
   ShowLegend: string,
   LegendPosition: string,
   DefaultSortSectors: string,
+  ThresholdsMaxLimit: number
```

Comment:
Can we move this to Gauge chart parameters as it is not used in any other charts?

Reply:
sure